cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -454,6 +472,46 @@ func generateMountsCommonInstallRecover(mst *initramfsMountsState) error {
 	return sysconfig.ConfigureTargetSystem(configOpts)
 }
 
+func maybeMountSave(disk disks.Disk, encrypted bool, mountOpts *systemdMountOptions) (haveSave bool, err error) {
+	var saveDevice string
+	saveKey := filepath.Join(boot.InitramfsEncryptionKeyDir, "ubuntu-save.key")
```

Comment:
Learned the hard way that this should be `filepath.Join(dirs.SnapFDEDirUnder(boot.InitramfsWritableDir), "ubuntu-save.key")` for run mode, and `..dirs.SnapFDEDirUnder(boot.InitramfsHostUbuntuDataDir)...` for recovery.

Reply:
This is done now.